applications/MPMApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -65,11 +65,18 @@ namespace Python{
             rBackgroundGridModelPart, rInitialModelPart, rMPMModelPart);
     }
 
+    void GenerateLagrangeNodes(
+        ModelPart& rBackgroundGridModelPart)
+    {
+        MaterialPointGeneratorUtility::GenerateLagrangeNodes(rBackgroundGridModelPart);
+    }
+
     void  AddCustomUtilitiesToPython(pybind11::module& m)
     {
         m.def("SearchElement", SearchElementAccordingToDimension);
         m.def("GenerateMaterialPointElement", GenerateMaterialPointElementAccordingToDimension);
         m.def("GenerateMaterialPointCondition", GenerateMaterialPointConditionAccordingToDimension);
+        m.def("GenerateLagrangeNodes", GenerateLagrangeNodes);
```

Comment:
```suggestion
        m.def("GenerateMaterialPointCondition", GenerateMaterialPointConditionAccordingToDimension);
        m.def("GenerateLagrangeNodes", MaterialPointGeneratorUtility::GenerateLagrangeNodes);
```

It should be sufficient to do this without defining the function `GenerateLagrangeNodes` in lines 68-73

Reply:
ture, I modified it accordingly. Thanks.